tests/fixtures/sample_lib/modules/sample_task/streaming_module.py

```diff
@@ -0,0 +1,50 @@
+"""
+A sample module for sample things!
```

Comment:
maybe to distinguish from the other module? could probably be more nuanced
```suggestion
A streaming module for streaming things!
```